### DISCUSSION TITLE
What's the best way to name sites in a multi-language build?

### DISCUSSION THREAD

**Discussion by RyanRoberts** [craftcms/cms#10910] [Q&A] [answered]
I'm creating my first multi-language Craft site at the moment, English and Dutch, how do people name "sites" when it comes to multiple languages? Here's why I'm asking…

Normally I use `{{ siteName }}` throughout my Twig templates to output the company/site name and I've never thought much about it. But now this is slightly confusing within the control panel because the site selection fields show the same site name (although one character different due to the language). I thought of changing the site name to "English" and "Dutch" which made it clear in the control panel site select fields but obviously looked stupid on the frontend and in some places within Craft.

Is there a preferred way to do this and make it clear where the client is in the CMS? Should I move what I use the siteName for (company name) into a global?

**Comment by mmikkel** [comment, marked as answer]
For multi-site installs, I give the sites names that make sense to content authors within the Craft control panel, and either hard code the front-facing site/company name in the templates, or use the `systemName` global variable. The latter can be edited via Settings -> General.

**Reply by RyanRoberts** [reply]
Thanks, that's kinda what I was thinking but I wasn't too sure. I'll look at using systemName too!
